### DISCUSSION TITLE
Campaign Deleted

### DISCUSSION THREAD

**Discussion by FlobbyDobby** [Lurkars/gloomhavensecretariat#503] [Q&A]
I have been using the windows/browser version of the app over the last few weeks,

My campaign was deleted, and settings reset after closing the app, 

I have not cleared cookies or otherwise,

Is there any ways to retrieve the data of my lost campaign? I did not make backup or manually save the game

**Comment by Lurkars** [comment]
I am sorry. I still have no clue how our when this happen. I only got two reports about this data loss before, but this was exactly the reason to add automatic backup settings and a backup reminder. If you don't have any, I have no idea how to restore the data. For future use I propose to create backups and use a server connection up also backup the latest state.
